src/torrent.cpp

```diff
@@ -9227,22 +9227,31 @@ namespace libtorrent
 
 	void torrent::queue_up()
 	{
+		// fix async position change calls for race conditions on changed torrents
+		if(!m_auto_managed || m_abort || is_finished()) return;
```

Comment:
@Falcosc do you remember why ``m_auto_managed`` is part of this condition? This is the thing that prevents paused torrents (or any non-auto-managed torrent) from having its queue position updated. https://github.com/arvidn/libtorrent/issues/2097